test/package.json

```diff
@@ -2,12 +2,16 @@
   "name": "stencil-test",
   "private": true,
   "scripts": {
-    "analysis": "node ./.scripts/analysis.js",
-    "build": "npm run build.hello-world && npm run build.hello-vdom && npm run build.todo && npm run build.end-to-end && npm run build.ionic && npm run analysis",
+    "TODO-STENCIL-389": "echo Remove the exit call below",
+    "analysis": "exit 0 && node ./.scripts/analysis.js",
```

Comment:
This command (and `build.ionic` below) can have `exit 0` here as they're run in a subshell, and therefore don't cause premature ending of the parent script